docs/dev/table/sql/gettingStarted.md

```diff
@@ -0,0 +1,200 @@
+
+In Flink SQL semantics, source and sink will be tables, but we know Flink isn’t a storage engine and cannot store the data. So we need to back our table with a [storage connector]({{ site.baseurl }}/dev/table/connect.html) like [file system]({{ site.baseurl }}/dev/table/connect.html#file-system-connector), [Kafka]({{ site.baseurl }}/dev/table/connect.html#kafka-connector) or [MySQL]({{ site.baseurl }}/dev/table/connect.html#jdbc-connector). When we will be defining these tables, we will configure the storage connector type, [format]({{ site.baseurl }}/dev/table/connect.html#table-formats) and schema for each table. 
+
+ 
+### Input Source Tables
+SQL API environment is configured using the YAML[yaml.org] configuration files. When we start SQL client, it reads the default configuration from the `/conf/sql-client-defaults.yaml` but it can be overriden by user defined configuration file. These files are used to define different environment variables including table source, sinks, [catalogs](({{ site.baseurl }}/dev/table/catalogs.html)), [user-defined functions](({{ site.baseurl }}/dev/table/functions/udfs.html)).
+
+Tables can be defined through the environment config file or using the SQL Client. Environment file will have YAML format while SQL client will use [SQL DDL commands](({{ site.baseurl }}/dev/table/sql)). 
+
+Following is an example to define a source table using file system with [csv format]({{ site.baseurl }}/dev/table/connectors/formats/csv.html) in the environment config file but Flink community has added the support for quite a few [formats]({{ site.baseurl }}/dev/table/connectors/formats/). 
+
+{% highlight yaml %}
+
+tables:
+  - name: EmployeeTableSource
+    type: source-table
+    update-mode: append
+    connector:
+      type: filesystem
+      path: "/path/to/something.csv"
+    format:
+      type: csv
+      fields:
+        - name: EmpId
+          data-type: INT
+        - name: EmpName
+          data-type: VARCHAR
+        - name: DeptId
+          data-type: INT          
+      line-delimiter: "\n"
+      comment-prefix: "#"
+    schema:
+      - name: EmpId
+        data-type: INT
+      - name: EmpName
+        data-type: VARCHAR
+      - name: DeptId
+        data-type: INT
+{% endhighlight %} 
+
+Addittionally, we can use SQL DDL to [create]({{ site.baseurl }}/dev/table/sql/create.html), [alter]({{ site.baseurl }}/dev/table/sql/alter.html), [drop]({{ site.baseurl }}/dev/table/sql/drop.html) tables from the SQL client. Same table can be defined using DDL as follows
+
+{% highlight sql %}
+CREATE TABLE EmployeeTableSource (
```

Comment:
Can we reorder this, lets show off create table first and then mention  yaml as an alternative.